dango/types/src/oracle/wormhole.rs

```diff
@@ -0,0 +1,254 @@
+        f.write_str("wormhole-vaa")
+    }
+
+    fn visit_str<E>(self, v: &str) -> Result<Self::Value, E>
+    where
+        E: serde::de::Error,
+    {
+        WormholeVAA::from_str(v).map_err(E::custom)
+    }
+}
+
+#[cfg(test)]
+pub use tests::*;
+
+#[cfg(test)]
+mod tests {
+
+    use {super::*, grug::MockStorage};
+
+    pub const VAA: &str = "UE5BVQEAAAADuAEAAAAEDQBkMyJzGWOwAlhd3NDvcYJvct5KACRi6oi9InIE/PYqXh1z92MOXFyFPGP5y9uOpubgMIvUh/pa5aXsM/z+aaCdAALKQlwSVB5YIQ/C0NuqXqam0fAAQYUJeBe+G7rjnv7UXhHRIqNiqCvTE1ygz3zUztg07pqoYahCI7SlqI23hHizAAPG7cQdoENAUMDgYC1znnRkG8NUDS/Yzlxb3Krl/fKDUjpgKM2ZEB5HD11bCTzIhPHTI8KQxIDbyKxF6o4cwf5QAAQxrIWXQX0Bx9/lDEDfFOOqRU6LwZhFMmiDwUedUxsIvR73V/yfZKNtObHA0O9McjdTo1JibRqnbNqw6H8hw4/JAAax4DOJ/M8yxbIk88rV0n8sttzelXPuMnnJCXV2CFpwlSqYu0cQ+gmWvfjK/zJSFKHhNF0N7wzOX9J/bghUeQ8nAQgJ7BPYtJo/qowTuQfDCa4ZHIhLjC9frRQh3/UWLrxosG5xWODfYWtpDLKwfmi2gjMV4PIMUdhwZLyMDfZIqR6MAQrB/IQ438iz+1cgU+i8ij7eB5+MeUxcV0ukQhJW/0nwVCm234OqZ+ES3fNPIpWHRo4nq5ZVCdX4ZE3MF+SjZIW2AAu4DFxPpw3tokuOP6z2jNk9AFzjC/WUqlZaIx+6Se5ZeGr4chhEh2IiwChhSUJnGsKtkXHSqTuLZpXf8QZ+ZiRFAAz9XiWxbiOvw6E4+I/0JRutYrALssiRNYBah4I1QzYSU1gIAeMEHz2jvMX9lGGZMfS/uJrv1VtW9UCJMxMCUqgOAA2Hkv95hjyj6toIigG6PyEpzzoJE3ZVqI92F2kWoGSE0l/7aV/sz6jhRl8udbq/Mqu+i9wpbUZqa/ZUCFFi0NLSAQ5s3Le7hPfK1QnMOU8eWkJqiy/XL+remqBwR92Omm8FFANUVzHwOKBsj0Zlrp9o7UW05BJUrUgVXbvJ61r2F+zoAREVSnZt5Tt3JOQs/JRFUway6AvKiQQJihLAOo6AkKiUCTR2G4kbFGiILq4hwgASZGshfdgKRCy+jbHlfDGpNF+vABIwoeTGgkil6kOH/Dg+hNKmqS8N41Y1tQn7i7RkfjMw7gMOQoZcNTKDCNGfgR0gu62ZIkDBIXmea25leCk6VnH2AGcgG4EAAAAAABrhAfrtrFhR4yubI7X5QRqMK6xKrj7U3XuBHdGnLqSqcQAAAAAFVzmdAUFVV1YAAAAAAApj+2QAACcQuyA5y12P+HQ9xkG4YvVJJeqDZf4BAFUAydiwdaXGkwM2WuI2M9TghRmb9cUgo7kP7RMioDQv/DMAAAZaLZ4aygAAAAIyAxQV////+AAAAABnIBuBAAAAAGcgG4AAAAZXwuHPYAAAAAJwWNtUCsIlij3mTR7FLM4Pu9qzDhJrUtUxIctFWnmj84Af485oCfcURBzjS8v9xlCaHMjofeED+Ml66aUMg3GKE8PDVhr5SAP4MJU436Fr6IFOxCWwq4hIuPuRgtLh6xy3t1dAZmA1SLzhr+OAOS1cKUapaSIeOdv/Mclu2fbSsnRU72f3eNeVU1v13bHKNJ70zxX/fMj109FD2kNQf4+VnjXn0jbxUKWfH5PZBT9oXoD9C59CFRYhLKAuMLSgi1sRBH0T1SmF59vcZjsn";
+
+    pub const GUARDIAN_SETS: Map<u32, GuardianSetInfo> = Map::new("guardian_sets");
+
+    pub const GUARDIANS_ADDRESSES: [&str; 19] = [
+        "WJO1p2w/c5ZFZIiFvczAbNcKPNM=",
+        "/2y5Ulib3oYsJe9DkhMvudSkIVc=",
+        "EU3oRgGTvfOi/PgfhqCXZfR2L9E=",
+        "EHoAhrMtegl3kmogUTHYcx05y+s=",
+        "jIKy/YL67ScR1Zrw8kmdFucm9rI=",
+        "EbOXVsBCRBvm2GULabVOvnFeI0M=",
+        "VM5bTTSPt0uVjolm4uw9vUlYp80=",
+        "FefK8HxOPcjnxGn5LIzYj7gAWiA=",
+        "dKO/kTlT1pUmDYi8GqJaTu42PvA=",
+        "AArAB2cns1++otrCj+5cyw/qdo4=",
+        "r0XO0Ta52eJJA0ZK6In1yKcj/BQ=",
+        "+TEkt8c4hDy7iehkyGLDjN3Mz5U=",
+        "0sw3pNwDao0jK0j2LN1HMUEvSJA=",
+        "2nmPaJajMx9ktIwS0dV/2cvnCBE=",
+        "caob4dNsr+OGeRD5nAnjR4mcGcM=",
+        "gZK25zh8zXaCd8F9qxt6UCfAs88=",
+        "F44hrS53rgZxFUnPux+cep2Alug=",
+        "XhSH81UV0CqSdTUEqNdUcbn0nts=",
+        "b768iY9APkdz6V/rFegMmpnINI0=",
+    ];
+
+    pub const GUARDIAN_SETS_INDEX: u32 = 4;
```

Comment:
is this duplicate with the `GUARDIAN_ADDRESSES` in dango-genesis?

if yes, why not export this, and in dango-genesis do `use dango_types::GUARDIAN_ADDRESSES`?